compiler-rt/lib/sanitizer_common/sanitizer_libc.cpp

```diff
@@ -279,6 +279,25 @@ uptr internal_wcsnlen(const wchar_t *s, uptr maxlen) {
   return i;
 }
 
+wchar_t* internal_wcscpy(wchar_t* dst, const wchar_t* src) {
+  wchar_t* dst_it = dst;
+  do {
+    *dst_it++ = *src++;
+  } while (*src);
+  return dst;
+}
+
+wchar_t* internal_wcsncpy(wchar_t* dst, const wchar_t* src, uptr maxlen) {
```

Comment:
same

Reply:
It would be nice if you have internal_* related changes in a separate patch, so I can approve them.
I would like to leave Win32 stuff to other folks.

Reply:
I can certainly do that.

Reply:
@vitalybuka opened this as a PR, https://github.com/llvm/llvm-project/pull/66529